src/lib/canvas/utils/canvas.js

```diff
@@ -0,0 +1,19 @@
+export const mountOnDragstartEventHandler = (parentId, callback) => {
+  const list = document.getElementById(parentId);
+
+  if (list) {
+    for (const el of list.children) {
+      el.addEventListener('dragstart', callback);
+    }
+  }
+}
+
+export const unmountOnDragstartEventHandler = (parentId, callback) => {
+  const list = document.getElementById(parentId);
```

Comment:
the logic for both methods are very simple, put it in a common function with an arg **action** = 'add' | ''remove